docs/source/1.0/guides/converting-to-openapi.rst

```diff
@@ -518,6 +518,34 @@ useIntegerType (``boolean``)
             }
         }
 
+.. _generate-openapi-setting-onErrorStatusConflict:
+
+onErrorStatusConflict (``String``)
+    Specifies how to resolve multiple error responses that share the same HTTP status code.
+    This behavior can be customized using the following values for the ``onErrorStatusConflict`` setting:
+
+    * USE_ONEOF
+    Use OpenAPI's ``oneOf`` keyword to combine schemas of errors with same HTTP status code.
+
+    .. note::
+            ``oneOf`` keyword is not supported by Amazon API Gateway.
+
+    * USE_PROPERTIES
+    Use ``properties`` field of OpenAPI schema object to combine schemas of errors with same HTTP status code.
```

Comment:
Can you expand here? All of the errors that share a status code are merged together, right? What happens if properties conflict? Or if they're required in one error but not another?

Reply:
Resolved. When they conflict, exception is thrown. Also, the `@required` trait will be dropped from all member shapes within conflicting error structure shapes.